scraper.py

```diff
@@ -1,46 +1,73 @@
 from selenium import webdriver
 import time
 import sys
+from selenium.webdriver.common.by import By
+
+driver = webdriver.Firefox()
```

Comment:
do we need to update the `Readme.md`?

Reply:
if we replace Firefox driver with Chrome , then we dont need to change Read.md

Reply:
If using Firefox doesn't require an additional web driver, then it's good. I've approved, you can merge by yourself now. Thank you

Reply:
thanks for your approval but why the code dosent merge?

Reply:
Merged.

I thought the requester could merge the PR by themselves once approved.